.github/workflows/deploy_storybook.yml

```diff
@@ -57,17 +57,19 @@ jobs:
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
 
-      - name: Set Docker context
-        uses: arwynfr/actions-docker-context@v2
+      - name: Deploy Serverless Container Scaleway
+        uses: philibea/scaleway-containers-deploy@v1.0.3
```

Comment:
Should it be in scw group ?

Reply:
It could be nice to change the prefix in the action options ?

Reply:
We can easily migrate this github action after if necessary.

Reply:
Which prefix ?